en.lproj/Localizable.strings

```diff
@@ -3676,6 +3676,306 @@
 "GrilledPorkSpringRoll.HowTo.2.Accessibility" = "For the peanut sauce, add some Sriracha sauce if you like spicy food.";
 
 
+// MARK: Bo Bia (Jicama Spring Roll) Page
+
+/* Jicama Spring Roll Introduction 1 */
+"JicamaSpringRoll.Introduction.1" = "##Bò Bía## (jicama spring roll) is another variation of ##Gỏi Cuốn## (spring roll) that is slightly different from other ##Gỏi Cuốn## (spring rolls) because it uses jicama and carrot instead of vermicelli noodles. The jicama and carrot is stir-fried and seasoned, which contributes a lot of flavor to the ##Gỏi Cuốn## (spring roll). ##Bò Bía## (jicama spring roll) is actually my favorite type of ##Gỏi Cuốn## (spring roll). However it is not sold in most Vietnamese restaurants so I would always have to make it myself whenever I have a craving, but it’s all worth the effort because it’s delicious.";
```

Comment:
```suggestion
"JicamaSpringRoll.Introduction.1" = "##Bò Bía## (jicama spring roll) is another variation of ##Gỏi Cuốn## (spring roll) that is slightly different from other ##Gỏi Cuốn## (spring rolls) because it uses jicama and carrot instead of vermicelli noodles. The jicama and carrot is stir-fried and seasoned, which contributes a lot of flavor to the ##Gỏi Cuốn## (spring roll). ##Bò Bía## (jicama spring roll) is actually my favorite type of ##Gỏi Cuốn## (spring roll). However, it is not sold in most Vietnamese restaurants so I would always have to make it myself whenever I started to crave it, but it’s all worth the effort because it’s delicious.";
```